sections/java.zsh

```diff
@@ -24,7 +24,7 @@ spaceship_java() {
 
   spaceship::exists java || return
 
-  local java_version=$(java -version 2>&1 | head -n 1 | awk -F '"' '{print $2}')
+  local java_version=$(java -version 2>&1 | grep version | awk -F '"' '{print $2}')
```

Comment:
We have a utility for grep `spaceship::grep`. It has some additional safeguards. Could you check if using that utility works for you?

Reply:
I tried without success nor errors the following:
```
local java_version=$(java -version 2>&1 | spaceship::grep version | awk -F '"' '{print $2}')
```
Am I applying it right?

Reply:
yes, you are. it should work. @Runrioter could you help with that?

Reply:
I was on v4.2.5, I updated everything, it is good now.